servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -8881,7 +8881,7 @@ void validateRoleMetaAuditEnabledFlag(RoleMeta meta, Role role, Domain domain, f
             if (domain.getAuditEnabled() != Boolean.TRUE) {
                 throw ZMSUtils.requestError("Role cannot be set as audit-enabled if the domain is not audit-enabled", caller);
             }
-            if (role.getAuditEnabled() != Boolean.TRUE && !role.getRoleMembers().isEmpty()) {
+            if (role.getAuditEnabled() != Boolean.TRUE && role.getRoleMembers() != null && !role.getRoleMembers().isEmpty()) {
```

Comment:
please make sure to add a unit test case for this function that checks for the null role members object.

Reply:
Hey @havetisyan thank you for the heads up!
I went ahead and fixed two issues I found (this time with groups) and added tests for them as well.